### ISSUE TITLE
Meeting Notes 2020-08-05

### ISSUE THREAD

**Issue by cessda-bitbucket-importer** [cessda/cessda.cmv#69]
**Original report on BitBucket by Wolfgang Zenk-Möltgen.**

----------------------------------------

### **Participants**

* @&zwnj;Kerrin Borschewski @&zwnj;Darren Bell @&zwnj;jeanninebeeken @&zwnj;TainaFSD @&zwnj;Wolfgang Zenk-Möltgen

### **Agenda**

1. Close Meeting Notes 2020-07-22 ~~#66~~

    1. Alex will work on last week in August to finish remaining issues, so deadline for finished tool will be ok
    2. remaining action items have a new sequence below:
    3. @&zwnj;Darren Bell will contact Jon on Friday about possible collaboration
    
2. Current issues

    1. CMV Tool: components available at #67
    
        1. UI: [https://cmv-dev.cessda.eu/ui](https://cmv-dev.cessda.eu/ui)
        2. Documentation: [https://cmv-dev.cessda.eu/user-documentation/constraint-catalogue.html](https://cmv-dev.cessda.eu/user-documentation/constraint-catalogue.html)
        3. User: cessda Password: CESSDA2020
        
    2. Constraints
    
        1. Validation Gates require a new combination, so that current CDC use case is matched: ELSST and CESSDA TC have no code, so checking of descriptiveTerm is necessary to have in BasicPlus Validation Gate. Also, mandatory and mandatory if parent element is present is not a distinction that users make or CMM has defined. The group agrees to have
        
            1. Basic: mandatory ~~#10~~ and mandatory if parent ~~#22~~
            2. BasicPlus: CV code ~~#9~~ and CV descriptive Term #27
            3. Standard: recommended #~~#11~~
            4. Extended and Strict stay as they were defined in ~~#23~~
            
        2. Alex asked not to open ~~#23~~, so we will make a new issue for that, todo @&zwnj;Wolfgang Zenk-Möltgen \(done #70\)
        
    3. DDI Profiles
    
        1. Implementation will happen first \(see action items, next week\)
        2. They should be included by Alex on the development server for testing \(end of August\)
        3. Testing by SPs and CDC user group can happen afterwards \(September\)
        
    4. 
    
3. AOB

    1. Next week: no normal meeting, but extra profile meeting, at usual time/link \(@&zwnj;alexander-muehlbauer still on holiday, @&zwnj;Wolfgang Zenk-Möltgen on workshop\)
    

### **Action Items**

With a new sequence:

1\) create version 1 of the profile:

* @&zwnj;Darren Bell and @&zwnj;TainaFSD and @&zwnj;jeanninebeeken will solve issues from the feedback during next week
* @&zwnj;Darren Bell will create profile v1 afterwards

2\) @&zwnj;Wolfgang Zenk-Möltgen and @&zwnj;Kerrin Borschewski will create a short information text for users to test the tool/profile

3\)@&zwnj;TainaFSD and @&zwnj;jeanninebeeken to contact CDC user group for the tests afterwards

4\) @&zwnj;Kerrin Borschewski contact all SPs \(in parallel to 3\) for testing

‌

‌
